news/2018-06-12-mappers-choice-awards-2017-results.md

```diff
@@ -0,0 +1,353 @@
+
+[![](/wiki/shared/news/2018-06-12-mappers-choice-awards-2017-results/dsco.jpg)](https://osu.ppy.sh/u/4330511)
+
+A quick glance at his maps, and even his oldest ones would probably make one doubt just how much of a rookie he is. However, **[dsco](https://osu.ppy.sh/u/4330511)**'s just that more cultured that everyone else. He has brought a variety of styles and concepts to the table, and not just in terms of maps!
+
+<a name="taiko" id="taiko">![](/wiki/shared/news/2018-06-12-mappers-choice-awards-2017-results/mode_taiko.jpg)</a>
+
+[![](/wiki/shared/news/2018-06-12-mappers-choice-awards-2017-results/encounter.jpg)](https://osu.ppy.sh/s/474003)
+
+A perfect representation of following the pace of the song and respecting both flow and rhythm, [**encounter**](https://osu.ppy.sh/s/474003) by **[tasuke912](https://osu.ppy.sh/u/2774767)** is easily one of his greatest maps to date, and perhaps even his best ranked map so far.
+
+With the absence of any real SV, 1/6 snapping patterns and a strong emphasis on keeping the standard flow and rhythm of taiko play, **encounter**  represents a purist approach towards osu!taiko mapping that just simply *works*, and is something that every taiko player should try at least once.
+
+*written by [Arrival](https://osu.ppy.sh/u/1694000)*
+
+[![](/wiki/shared/news/2018-06-12-mappers-choice-awards-2017-results/next-level.jpg)](https://osu.ppy.sh/s/623246)
+
+Winning best marathon and electronic map as well as capturing runner-up for Grand Award, **[MMzz](https://osu.ppy.sh/u/128993)**'s map of **[NEXT LEVEL](https://osu.ppy.sh/s/623246)** has universally won the respect of taiko mappers and players alike. One only has to play the map once to understand why — **[MMzz](https://osu.ppy.sh/u/128993)** captures all the diversity offered by sky_delta and lapix's composition and transforms it into a cohesive playing experience. The relatively lower BPM of the map allows for liberal use of mixed-snapping patterns that play smoothly, keeping the map engaging from start to end.
+
+*written by [hikikochan](https://osu.ppy.sh/u/6512678)*
+
+[![](/wiki/shared/news/2018-06-12-mappers-choice-awards-2017-results/matsuri.jpg)](https://osu.ppy.sh/s/511420)
+
+A well rounded map completely deserving this place on the best spread of 2017. The song's characteristics gave the possibility of creating exotic lower difficulties using blue ticks efficiently. From Kantan to Inner Oni, these rhythmical particularities are well represented, giving both new and seasoned players a good challenge. One of the sets I'd recommend to new players starting taiko, the song and difficulties really sets it apart from the other ranked maps.
```

Comment:
really sets -> really set
set is the plural form, it's referring to song + diff

Reply:
well rounded -> well-rounded

Reply:
giving both -> offering both
both work, just adding variety since repetitive